bridge/contracts/libraries/StakingNFT/StakingNFT.sol

```diff
@@ -102,6 +103,10 @@ abstract contract StakingNFT is
         uint256 tokenID_,
         uint256 lockDuration_
     ) public override withCircuitBreaker onlyGovernance returns (uint256) {
+        require(
```

Comment:
Possibly. I am not sure.

I am including this because I thought it was a good idea to be explicit as to why something fails and keeping it local. In this case, the error is clearly because of tokenID not existing.

Reply:
Using a modifier actually increased the code size. So, it may not be a good idea.